src/emr_cluster.go

```diff
@@ -263,6 +281,37 @@ func (ec EmrCluster) GetInstanceGroups() []*emr.InstanceGroupConfig {
 	return instanceGroups
 }
 
+// GetEbsConfiguration turns a EbsConfigurationRecord into an emr.EbsConfiguration
+func GetEbsConfiguration(c *EbsConfigurationRecord) *emr.EbsConfiguration {
+	configs := c.EbsBlockDeviceConfigs
+
+	var emrConfigsArr []*emr.EbsBlockDeviceConfig
+
+	if configs != nil && len(configs) > 0 {
+		emrConfigsArr = make([]*emr.EbsBlockDeviceConfig, len(configs))
+
+		for i, config := range configs {
+			emrVolumeSpec := emr.VolumeSpecification{
+				Iops:       aws.Int64(config.VolumeSpecification.Iops),
+				SizeInGB:   aws.Int64(config.VolumeSpecification.SizeInGB),
+				VolumeType: aws.String(config.VolumeSpecification.VolumeType),
```

Comment:
I was on the fence about adding these kinds of validations, but I guess it can't hurt.